src/Kernel32.Shared/Kernel32Extensions.cs

```diff
@@ -19,14 +19,15 @@ public static partial class Kernel32Extensions
         /// Gets the text associated with a <see cref="Win32ErrorCode"/>.
         /// </summary>
         /// <param name="error">The error code.</param>
+        /// <param name="dwLanguageId">The language identifier for the message</param>
         /// <returns>The error message. Or <c>null</c> if no message could be found.</returns>
-        public static unsafe string GetMessage(this Win32ErrorCode error)
+        public static unsafe string GetMessage(this Win32ErrorCode error, int dwLanguageId = 0)
```

Comment:
I wonder if for each new public API (that isn't a p/invoke method direclty) that accepts lcid, we should make an overload that accepts the much more familiar `CultureInfo`.